package.json

```diff
@@ -32,7 +32,6 @@
         "url": "git+https://github.com/Ultimaker/react-web-components.git"
     },
     "author": "Ultimaker",
-    "license": "ISC",
```

Comment:
this repo is (deprectated but) open source, so we'll need to keep a license agreement in place.

Reply:
We might just have to leave this as-is, even though we don't have a copy of the license or any other mention elsewhere, just these three letters.

I don't think the license question was properly asked when this repo was set up either.

Reply:
Also, personally I would prefer to use MIT here, given that it is far more common while being functionally the same.

Reply:
^Agree